src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -56,59 +86,137 @@ export const PaymentContent: FC<Props> = props => {
+        <Media greaterThan="xs">
+          <ContinueButton onClick={onContinue} loading={isLoading} />
+        </Media>
+      </Collapse>
+
+      {/* Bank debit */}
+      <Collapse open={currentPaymentMethod === PaymentMethods.BankDebit}>
+        <Text color="black60" variant="sm">
+          • Bank transfer is powered by Stripe.
+        </Text>
+        <Text color="black60" variant="sm">
+          • Search for your bank institution or select from the options below.
+        </Text>
+        <Text color="black60" variant="sm">
+          • If you can not find your bank, please check your spelling or choose
+          another payment method.
+        </Text>
         <Spacer mb={2} />
-        <Collapse open={paymentMethod === "credit_card"}>
-          <PaymentPickerFragmentContainer
-            commitMutation={commitMutation}
-            me={me}
-            order={order}
-            innerRef={paymentPicker}
-          />
-          <Spacer mb={4} />
-          <Media greaterThan="xs">
-            <ContinueButton onClick={onContinue} loading={isLoading} />
-          </Media>
-        </Collapse>
-        <Collapse open={paymentMethod === "bank_debit"}>
-          <Text color="black60" variant="sm">
-            • Bank transfer is powered by Stripe.
-          </Text>
-          <Text color="black60" variant="sm">
-            • Search for your bank institution or select from the options below.
-          </Text>
-          <Text color="black60" variant="sm">
-            • If you can not find your bank, please check your spelling or
-            choose another payment method.
-          </Text>
-          <Spacer mb={2} />
-
-          <BankDebitProvider order={order} />
-        </Collapse>
-      </Flex>
-    </div>
+
+        <BankDebitProvider order={order} />
+      </Collapse>
+
+      {/* Wire transfer */}
+      <Collapse open={currentPaymentMethod === PaymentMethods.WireTransfer}>
+        <Text color="black60" variant="sm">
+          • To pay by wire transfer, complete checkout and one of our support
+          specialists will reach out with next steps.
+        </Text>
+        <Text color="black60" variant="sm">
+          • Your bank may charge a fee for the transaction.
+        </Text>
+        <Text color="black60" variant="sm">
+          • Questions? Email{" "}
+          <Clickable textDecoration="underline">orders@artsy.com</Clickable>
+        </Text>
+        <Spacer mb={4} />
+        <Media greaterThan="xs">
+          <Button
+            onClick={handleWireTransferSaveAndContinue}
+            variant="primaryBlack"
+            width="100%"
+          >
+            Save and Continue
+          </Button>
+        </Media>
+      </Collapse>
+    </Flex>
   )
 }
+
+/*
+returns all available payment methods, by checking relevant feature flags
+TODO: when ACH and wire_transfer FFs is removed, this function can be removed and radios can be moved to PaymentContent
+*/
+const getAvailablePaymentMethods = (
```

Comment:
The two-step "Payment Method" and Payment Details" will need to continue to be behind the feature flag until either ACH or wire transfer is released, so I'm sure if this would work in this case as it still shows the radio buttons when neither flag is enabled.

Reply:
Maybe we could have the third flag for "new payment methods" which job it to flag the two-step flow?

Reply:
Sorry second thought, we might need to check with a designer for this, but if the cases where there is only a single payment method available means there is no two-step process, we could just remove the first step for those cases.

Reply:
thank you for flagging this, I see what you mean. 

I think it makes sense to keep the original view (no collapse, just credit card), when neither of the other two is available. please let me know if you disagree.

Reply:
Yes, I agree!

Reply:
Yep sounds good to me!